### PR TITLE
Styling for orgs in list, adding pictures to token.

### PR DESCRIPTION
**Pull request by andrewconner**
@adamjgrant @cvializ 

The change:

### CODE REVIEW

File: packrat/styles/keeper/compose.less [hunk 1]
```diff
@@ -387,10 +387,31 @@ span.kifi-compose-tip-alt {
...
+      position: absolute;
+      top: 10px;
+      left: 10px;
+      height: 34px;
+      width: 34px;
+      background: 0 0/cover no-repeat;
+      box-shadow: 0 0 2px rgba(0,0,0,.2) inset;
       background-size: 20px auto;
       background-position: 7px 8px;
+      border-radius: 50% !important;
```
Comment (cvializ) [inline packrat/styles/keeper/compose.less]: the `!important`s make me sad, but the extension does some weird things with specificity, so if it needs them I will understand

**Comment by cvializ** [comment]
:+1:

**Comment by adamjgrant** [comment]
What @cvializ said, and :+1:

File: packrat/styles/keeper/compose.less (hunk 1 above)
Comment (andrewconner) [inline packrat/styles/keeper/compose.less]: Me too. It was there originally, and I needed to duplicate some code here, so I copied them. I'll be not lazy and test it out without and see what breaks.

Comment (andrewconner) [inline packrat/styles/keeper/compose.less]: You were right, I can't see a need. Removed.

**Comment by cvializ** [comment]
now I give :+1: :+1:

**Pull request merged**